components/Cards/DetailsCard.js

```diff
@@ -170,52 +298,146 @@ export default function DetailsCard({ id, isFavorite, toggleFavorite, place }) {
+                ))}
+              </StyledActivities>
+            </StyledFormGroup>
+          ) : (
+            <StyledCardSubHeading>
+              {region} &#183; &nbsp;
+              {activities.map((activity) => activity.activityname).join(", ")}
+            </StyledCardSubHeading>
+          )}
+          {editingPlaceId ? (
+            <StyledFormGroup>
+              <StyledFormLabel htmlFor="region">Region:</StyledFormLabel>
+              <select name="region" id="region" required>
+                {getUniqueValues("region").map((region, id) => (
+                  <option key={id} value={region}>
+                    {region}
+                  </option>
+                ))}
+              </select>
+            </StyledFormGroup>
+          ) : null}
           <StyledCardSubHeadings>Description:</StyledCardSubHeadings>
-          <StyledCardText>{description}</StyledCardText>
+          {editingPlaceId ? (
+            <StyledFormGroup>
+              <StyledFormLabel htmlFor="description"></StyledFormLabel>
+              <textarea
+                defaultValue={description}
+                id="description"
+                name="description"
+                rows="8"
+                cols="50"
+                required
+              ></textarea>
+            </StyledFormGroup>
+          ) : (
+            <StyledCardText>{description}</StyledCardText>
+          )}
           <StyledCardSubHeadings>Reviews:</StyledCardSubHeadings>
-          <StyledCardText>{initialReview}</StyledCardText>
+          {editingPlaceId ? (
+            <StyledFormGroup>
+              <StyledFormLabel htmlFor="initialReview"></StyledFormLabel>
+              <textarea
+                defaultValue={initialReview}
+                id="initialReview"
+                name="initialReview"
+                rows="8"
+                cols="50"
+                required
+              ></textarea>
+            </StyledFormGroup>
+          ) : (
+            <StyledCardText>{initialReview}</StyledCardText>
+          )}
+
           {session.status === "authenticated" &&
             userId === session.data.user.id && (
-              <StyledCardButtonWrapper>
-                <StyledSecondaryButton onClick={handleDeletePlace}>
-                  <FontAwesomeIcon icon={faTrashSolid} />
-                  &nbsp;Delete Place
-                </StyledSecondaryButton>
-                <StyledSecondaryButton>
-                  <FontAwesomeIcon icon={faPencilSolid} />
-                  &nbsp;Edit Place
-                </StyledSecondaryButton>
-              </StyledCardButtonWrapper>
+              <>
+                <StyledCardButtonWrapper>
+                  {editingPlaceId ? (
+                    <StyledSecondaryButton onClick={cancelEditPlace}>
+                      <FontAwesomeIcon icon={faTrashSolid} />
+                      &nbsp;cancel edit
+                    </StyledSecondaryButton>
+                  ) : null}
```

Comment:
You could also write all of these checks like this:

```suggestion
                  {editingPlaceId && (
                    <StyledSecondaryButton onClick={cancelEditPlace}>
                      <FontAwesomeIcon icon={faTrashSolid} />
                      &nbsp;cancel edit
                    </StyledSecondaryButton>
                  )}
```

This is also applicable for the following lines of code